tfx_bsl/beam/run_inference_test.py

```diff
@@ -572,6 +581,56 @@ def test_request_body_with_binary_data(self):
         },
     ], result)
 
+  def test_request_serialized_example(self):
+    example = text_format.Parse(
+        """
+      features {
+        feature { key: "x_bytes" value { bytes_list { value: ["ASa8asdf"] }}}
+        feature { key: "x" value { bytes_list { value: "JLK7ljk3" }}}
+        feature { key: "y" value { int64_list { value: [1, 2] }}}
+      }
+      """, tf.train.Example())
+    inference_spec_type = model_spec_pb2.InferenceSpecType(
+      ai_platform_prediction_model_spec=model_spec_pb2.
+      AIPlatformPredictionModelSpec(
+        project_id='example',
+        model_name='example',
+        version_name='example',
```

Comment:
Very nit: can we use some different names, like 'test_project', 'test_model', etc?